src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,46 @@
+.nav {
+  margin-right: 50px;
+
+  &__list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__list-item {
+    display: inline-block;
+    position: relative;
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: $reset;
+    }
+  }
+
+  &__link {
+    display: inline-block;
+    padding: 23px 0;
+    font-family: inherit;
+    font-weight: $medium-font-weight;
+    font-size: $medium-font-size;
+    color: $color-main-dark;
+    text-transform: uppercase;
+    text-decoration: none;
+  }
+
+  &__link:hover,
+  .is-active {
+    color: $color-blue;
+  }
+
+  .is-active::before {
```

Comment:
Similarly, the `.is-active::before` class should also be nested within `&__link`. Otherwise, it won't be applied to the correct element.